Code/Tools/FBuild/FBuildApp/Main.cpp

```diff
@@ -604,3 +620,20 @@ int WrapperIntermediateProcess( const AString & args, const FBuildOptions & opti
 }
 
 //------------------------------------------------------------------------------
+// DisplayTargetsList
```

Comment:
Each entry in the hash table is a linked list. You'll need to walk the chain (m_Next) or you won't find all the nodes when there are collisions.
You'll likely need to move this function into the NodeGraph in order to be able to access the m_Next member of Node.